app/precompiled.py

```diff
@@ -110,20 +111,30 @@ def validate_pdf_document():
     if not encoded_string:
         abort(400)
 
+    invalid_pages = get_invalid_pages(BytesIO(encoded_string))
     data = {
-        'result': validate_document(BytesIO(encoded_string)),
+        'result': len(invalid_pages) == 0
     }
 
     if not generate_preview_pngs:
         return jsonify(data)
 
-    if not data['result']:
-        data['message'] = 'Content in this PDF is outside the printable area'
-        pages = overlay_template_areas(BytesIO(encoded_string), overlay=True)
+    file_data = rewrite_address_block(BytesIO(encoded_string))
```

Comment:
I don't think we should rewrite address in a file that's known to exceed printable area. The address logic relies on the page margins which are likely to be wrong in this case.

Reply:
after discussion, will revert change